pwndbg/gdblib/stack.py

```diff
@@ -136,3 +136,27 @@ def is_executable() -> bool:
             nx = True
 
     return not nx
+
+
+def find_addr_on_stack(addr):
```

Comment:
this method is basically copied from here https://github.com/pwndbg/pwndbg/blob/dev/pwndbg/commands/stack.py#L26

i wonder if both should use this centralised approach?

Reply:
i'll make another issue to centralise that in another change i think ( and i'll fix that edge case and note it in the centralisation issue )